docs-website/sidebars.js

```diff
@@ -498,6 +498,7 @@ module.exports = {
       "docs/CONTRIBUTING",
       "docs/links",
       "docs/rfc",
+      "docs/rfc/active/5818-serial-updates",
```

Comment:
If we have removed the RFC section, we can go ahead and remove this. Or we can simply ignore this in sidebars build